notebook/static/base/js/keyboard.js

```diff
@@ -225,6 +225,17 @@ define([
         }
         return dct;
     };
+    
+    ShortcutManager.prototype.get_shortcut_for_action_name = function(name){
```

Comment:
Can this be `getShortcut` or `get` ?  So then when it's called it looks like this `shortcutManager.getShortcut('bob')` or `shortcutManager.get('bob')` .